src/main/resources/swagger/agora.yaml

```diff
@@ -439,6 +440,12 @@ paths:
           description: Snapshot ID of method.
           required: true
           type: number
+        - name: payloadAsObject
+          in: query
+          description: Instead of returning a string under key payload, return a JSON object under key payloadObject
```

Comment:
Would be nice to have some more info saying that you can't have `payloadAsObject` and `onlyPayload` both set to true in the same request.

Reply:
how about return a 400 if both are specified?

Reply:
It's not possible for a user to do that in Swagger because the flags apply to different endpoints:

`payloadAsObject` is only available on
```
/api/v1/configurations/{namespace}/{name}/{snapshotId}
```
and `onlyPayload` is only available on
```
/api/v1/methods/{namespace}/{name}/{snapshotId}
```
A person writing their own script will not be limited by the Swagger interface, and so would hit the 400 instead.

Reply:
Ah - thank you!